Python/initconfig.c

```diff
@@ -2232,6 +2239,7 @@ config_parse_cmdline(PyConfig *config, PyWideStringList *warnoptions,
     const PyWideStringList *argv = &config->argv;
     int print_version = 0;
     const wchar_t* program = config->program_name;
+    const wchar_t* xoption_help;
```

Comment:
Moved this here to appease a warning.  I hope `const` is correct!

Reply:
Compiler Knows Best 😄